examples/android/app/src/main/proto/cubic.proto

```diff
@@ -0,0 +1,345 @@
+// Copyright (2019) Cobalt Speech and Language Inc.
```

Comment:
Just to confirm, we're okay with having the .proto file duplicated here? If we had some mechanism to put it here automatically, or instructions in the docs that it should be copied, the example app can take advantage of changes in the main proto file. Otherwise, SDK maintainers need to remember to update this copy too.

Reply:
Oh, that's a good point.  Perhaps we could check it in as a symlink?